src/commands/link.ts

```diff
@@ -58,13 +65,22 @@ export const data = async () =>
     .toJSON();
 
 export const call = async (interaction: CommandInteraction) => {
+  const maybeMention = (msg: string, target: User | null) =>
+    target
+      ? `${userMention(target.id)} ${msg}`
+      : `${msg}
+
+${italic(
+  `Note: to use this command in response to a user's query, specify the ${inlineCode(
+    "target"
+  )} option when invoking the command.`
+)}`;
```

Comment:
Don't you think this is still ugly?

Why not move the function out of `call`, and also move the note out.
```typescript
export const call = async (interaction: CommandInteraction) => {
  const topic = interaction.options.getString("topic", true);
  const target = interaction.options.getUser("target");
  const linkInfo = LINKS[topic];
  const message = `See ${hyperlink(linkInfo.description, hideLinkEmbed(linkInfo.url))}`;
  await interaction.reply({
    content: maybeMention(message, target),
    ephemeral: !target,
  });
};

const maybeMention = (msg: string, target: User | null) =>
  target ? `${userMention(target.id)} ${msg}` : `${msg}\n\n${italic(NOTE)}`;

const NOTE =
  "Note: to use this command in response to a user's query, specify the `target` option when invoking the command.";
```